apps/ad-campaigns/ad-css-animations/styles/fonts.css

```diff
@@ -0,0 +1,142 @@
+@font-face {
+    font-family: 'Poppins Regular';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Poppins Regular'), url('../assets/fonts/poppins/Poppins-Regular.woff') format('woff');
+    }
+    
+
+    @font-face {
```

Comment:
Are we using all of these font styles? Seems like we don't need to add all weights